ci/jjb/jobs/macros.yaml

```diff
@@ -52,6 +52,9 @@
     # This hostname can be connected from other open stack vms.
     builders:
         - shell: |
+            # Hostnames that are set
+            echo "$(hostname --all-fqdn)"
+            echo "$(hostname)"
```

Comment:
The `hostname` executable already prints to stdout. There's no benefit to capturing its output and then printing to stdout with `echo`.

Reply:
Hi @Ichimonji10 ,

The hostname which gets printed are ones which are filtered out by the `grep openstacklocal` command. The actual output which shows the `hostname --all-fqdn` is not seen in the Jenkins jobs. Let me know if I am off base here.

Thanks

Reply:
I think my point has been lost in translation. See: https://github.com/pulp/pulp-ci/pull/587